system/lib/wasmfs/wasmfs.cpp

```diff
@@ -22,9 +32,9 @@ namespace wasmfs {
 // system priority) since wasmFS is a system level component.
 // TODO: consider instead adding this in libc's startup code.
 // WARNING: Maintain # n + 1 "wasmfs.cpp" 3 where n = line number.
-# 26 "wasmfs.cpp" 3
+# 36 "wasmfs.cpp" 3
 __attribute__((init_priority(100))) WasmFS wasmFS;
-# 28 "wasmfs.cpp"
+# 38 "wasmfs.cpp"
```

Comment:
Maybe the new code added higher up could be added after this? Given how sensitive this is, putting this init priority stuff as near the top of the file as possible seems best.

Reply:
Sounds good :)